class NotFoundException(Exception):
    def __init__(self):
        pass

class AlreadyExistsException(Exception):
    def __init__(self):
        pass